docs/api-docs/cart-and-checkout/custom-order-confirmation.md

```diff
@@ -84,6 +89,10 @@ To install a custom order confirmation page on a store, follow these steps:
 3. Click the **Save** button at the bottom of the page.
 4. Navigate to your live storefront to view your new custom order confirmation page.
 
+### Installing using the Checkout v3 API
```

Comment:
```suggestion
### Install using the Checkout v3 API
```
style guide

Reply:
also, in many of the docs I've been working on, I see version before API name -- for example, "V3 Checkout API". With the V capitalized. Is either of these proscribed by the style guide, @traci707 / @markcmurphy

Reply:
@slsriehl I did not find information in the style guide but I like your suggestion and think it looks good.